plugins/README.md

```diff
@@ -34,10 +52,16 @@ type Price struct {
 // PluginState is the returned data when the oracle host want to initialise the plugin with basic information: version,
 // and available symbols that the data source support.
 type PluginState struct {
+	KeyRequired      bool
 	Version          string
 	AvailableSymbols []string
 }
 ```
+where the statements in PluginState:
+- KeyRequired states that if the plugin need a service key to be configured, for a key required plugin, oracle server will not start the plugin if a key is missing from configuration.
+- Version states the version of the plugin.
+- AvailableSymbols states the available symbols of current data plugin.
```

Comment:
I find this hard to understand because of 'available'.  What does this mean?

Does it mean the set of symbols the oracle server is configured to fetch? 

I.e.:

```suggestion
- AvailableSymbols states the set of symbols the data plugin is configured to fetch.
```